plugins/automation/spec/plugin_helper.rb

```diff
@@ -35,4 +35,14 @@ def self.capture
   triggerables [DiscourseAutomation::Triggers::API_CALL]
 end
 
-RSpec.configure { |config| config.include AutomationSpecHelpers }
+RSpec.configure do |config|
+  config.include AutomationSpecHelpers
+
+  config.before(:suite) do
+    if defined?(migrate_column_to_bigint)
+      migrate_column_to_bigint(DiscourseAutomation::Field, :automation_id)
+      migrate_column_to_bigint(DiscourseAutomation::PendingAutomation, :automation_id)
+      migrate_column_to_bigint(DiscourseAutomation::PendingPm, :automation_id)
```

Comment:
I'm 99.9% sure we can just migrate these to bigint since automation happens mostly in the background?

Reply:
I guess we can review this decision in the future instead of doing it now.